_includes/releases/v22.2/v22.2.0-alpha.1.md

```diff
@@ -0,0 +1,851 @@
+- Using [`COPY`](../{{site.versions["dev"]}}/copy-from.html) in the SQL shell is now supported while inside an explicit transaction. [#82101][#82101]
+- `CTRL+C` (the interrupt signal) can now be used in the CLI to attempt to cancel the currently executing SQL query. [#82101][#82101]
+- [`cockroach sql`](../{{site.versions["dev"]}}/cockroach-sql.html) (and thus [`cockroach demo`](../{{site.versions["dev"]}}/cockroach-demo.html) too) now support the client-side commands `\o` and `\qecho` like `psql`. The command `\o` can redirect the output of SQL queries to a file. `\qecho` adds an arbitrary text to the current query output file. [#83118][#83118]
+- CockroachDB now produces a clearer error when the path specified via `--socket-dir` is too long. [#84532][#84532]
+- When the flag `--background` is specified, CockroachDB now makes three attempts to find a suitable directory to create the notification socket: the value of `--socket-dir` if specified, the value of `$TMPDIR` (or `/tmp` if the environment variable is empty), and the current working directory. If none of these directories has a name short enough, an explanatory error is printed. [#84532][#84532]
+
+<h3 id="v22-2-0-alpha-1-api-endpoint-changes">API endpoint changes</h3>
+
+- Added logic to support dropping unused [index](../{{site.versions["dev"]}}/indexes.html) recommendations. [#77642][#77642]
+- `ListSessions` now returns closed sessions in addition to open sessions. `ListSessionsRequest` now has a `exclude_closed_sessions` flag, which is a `BOOL` to exclude closed sessions. `serverpb.Session` now has `end` and `status` fields, which specify the time the session ended and the status (`opened`, `closed`) of the session, respectively. [#78650][#78650]
+- Updated `api/v2/rules` endpoint to include alerts defined in our [documentation](../{{site.versions["dev"]}}/monitoring-and-alerting.html). [#80274][#80274]
+- The `ListSessions` API has a new field `last_auto_retry_reason` under the `active_txn` field for a session. This field contains the string describing the retry reason or `nil` if none exists. This is also surfaced in the [`crdb_internal.{cluster,node}_transactions`](../{{site.versions["dev"]}}/crdb-internal.html) tables and [`SHOW TRANSACTIONS`](../{{site.versions["dev"]}}/show-transactions.html) under the `last_auto_retry_reason` column. [#81531][#81531]
+- `serverpb.Session` now has three new fields: number of transactions executed, transaction fingerprint IDs, and total active time. [#82352][#82352]
+- Added information about total bytes, live (non MVCC) bytes and live (non MVCC) percentage to the table details endpoint. [#83677][#83677]
+- Added support for index recommendations to be returned on the statement details API. [#85863][#85863]
+
+<h3 id="v22-2-0-alpha-1-db-console-changes">DB Console changes</h3>
+
+- Added index created time as an option on the DB Console [Databases Table page](../{{site.versions["dev"]}}/ui-databases-page.html). [#78283][#78283]
```

Comment:
Page is just called `[Databases page]`

Reply:
Ah yes, realize it's the Database page, but this is under the Table View. Shall clarify